packages/cli/src/utils/index.js

```diff
@@ -110,3 +110,21 @@ export function showBanner(nuxt) {
 
   process.stdout.write(box + '\n')
 }
+
+/**
+ * Normalize string option in command
+ *
+ * @export
+ * @param {String} option
+ * @param {*} defaultValue
+ * @returns formatted option
+ */
+export function normalizeOption(option, defaultValue) {
```

Comment:
I would suggest renaming to `normalizeArg`

Reply:
ok